microsoft/PowerToys
# Delete Fody and PropertyChanged.Fody

## Summary of the Pull Request

**What is this about:**
Remove Fody and PropertyChanged.Fody. PropertyChanged.Fody makes modifications to IL code so we do not have to implement `INotifyPropertyChanged` for properties.

**What is include in the PR:** 
- Remove packages Fody and PropertyChanged.Fody
- Add `OnPropertyChanged` invocation for in code subscribed properties. For example
https://github.com/microsoft/PowerToys/blob/7514758b8c2b1f6dcd31546e891f0ae714d38017/src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs#L90-L107
- Add `OnPropertyChanged` invocation for properties that are used in xaml files

**How does someone test / validate:** 
- Check if the list of properties that must call `OnPropertyChanged` is complete. As was described before it is properties that are used in xaml or `OnPropertyChanged` subscriptions in code.
- Test for regression for PT Run. Do settings apply? Does navigation works?...

## Quality Checklist

- [X] **Linked issue:** #8638

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
